grouper/fe/handlers/service_account_view.py

```diff
@@ -51,7 +51,10 @@ def get(self, user_id=None, name=None):
         groups = [{'name': g.name, 'type': 'Group', 'role': ge._role} for g, ge in group_edge_list]
         public_keys = get_public_keys_of_user(self.session, user.id)
         permissions = user_md.get('permissions', [])
-        log_entries = get_log_entries_by_user(self.session, user) + group.my_log_entries()
+        # Combine the logs from the User and Group component. Use set to remove any duplicates
+        # and sorted to sort the combined logs by log time
+        log_entries = sorted(set(get_log_entries_by_user(self.session, user) +
+            group.my_log_entries()), key=lambda x: x.log_time, reverse=True)
```

Comment:
set uses __hash__ to determine identical values, so that will never happen (unless we overload __hash__ on AuditLog to be based solely on the log time, which we shouldn't)